### PR TITLE
snip pkg/util/strings dependency

### PR DESCRIPTION
**Pull request by deads2k**
The `pkg/util/strings` package looks to be largely used by volumes, which are independent of the bits used by genericapiserver which aren't used by anyone else.  This moves the single function (used no where else) to its point of use.

@sttts

### CODE REVIEW

**Comment by fejta** [comment]
@k8s-bot test this

kubernetes/test-infra#1486

**Comment by deads2k** [comment]
@k8s-bot test this

File: pkg/genericapiserver/api/installer.go [hunk 1]
```diff
@@ -1080,3 +1080,32 @@ func splitSubresource(path string) (string, string, error) {
 	}
 	return resource, subresource, nil
 }
+
+// lifted from k8s.io/kubernetes/pkg/util/strings to skip the link
```
Comment (sttts) [inline pkg/genericapiserver/api/installer.go]: no need for this comment.

**Comment by sttts** [comment]
one nit, otherwise lgtm

**Pull request merged**